src/app/content/constants.ts

```diff
@@ -16,3 +17,10 @@ export const highlightStyles: Array<{label: HighlightColorEnum, passive: string,
   {label: HighlightColorEnum.Purple, passive: '#cbcfff', focused: '#545ec8'},
   {label: HighlightColorEnum.Pink, passive: '#ffc5e1', focused: '#de017e'},
 ];
+
+export const practiceQuestionStyles: PracticeQuestionStyles[] = [
```

Comment:
this stuff doesn't belong in `app/content/constants.ts`, it should be under the PQ directory